src/OnboardingSPA/steps/SiteGen/Editor/Header/stylesheet.scss

```diff
@@ -160,3 +160,15 @@
 		}
 	}
 }
+
+body {
+
+	&.editor-actions-disabled {
+		pointer-events: none;
+		opacity: 0.7;
```

Comment:
Will this even fade out the Save and Continue button itself?

Reply:
yes this fades out the Save and Continue button button too but it is not visually opaque and achieves the effect or alternatively we can use an overlay.